### DISCUSSION TITLE
Prod Buld: Is possible to remove multiple __VP_HASH_MAP__ of the HTML generated and put in a single file?

### DISCUSSION THREAD

**Discussion by sr2ds** [vuejs/vitepress#3547] [Q&A] [answered]
Hello my friends,

My context is: I have a website with +10K posts.

For each html generated, we have the __VP_HASH_MAP__.

My goal is: Decrease the memory size of the assets generated (memory in the disk).

My idea is, maybe, if my html generated can to use the `script src=` for some repeated data, instead of repeat the same values inside the __VP_HASH_MAP__,  my html file will be a little bit minor.

I know that it can be hard and probably break the static generation concept, but I'm thinking some solutions and I'm thinking in this idea right now.

Thanks

**Comment by brc-dd** [comment, marked as answer]
Set metaChunk: true in vitepress config.

**Reply by sr2ds** [reply]
Super thanks, it is great. 
It solved my problem!! Wooow! Thanks
